src/Components/Cards/TextCard/component/CommentsTab.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from 'react'
+import { useSelector } from 'react-redux'
+import { useDispatch } from 'react-redux'
+import { addComment } from '../../../../store/postSlice'
+import './CommentsTab.css'
+
+export const CommentsTab = ({postId, setClicked}) => {
+    const [commentData, setCommentData]= useState({content:""})
+    const { token } = useSelector((store) => store.authenticate);
+    const dispatch = useDispatch()
+    // console.log("ID",postId)
+
+    const clickHandler = ()=>{
+        setClicked((prev)=>!prev)
+        dispatch(addComment({postId:postId, commentData:commentData, encodedToken:token}))
```

Comment:
Can directly use object property names while passing object as arg to `addComment()` here to make code more readable. 
```
{postId, commentData, encodedToken}
```